healthcheck/healthcheck.py

```diff
@@ -181,3 +182,27 @@ def _assess_overall_status(self):
             return False
 
         return True
+
+
+def _file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
+    """
+    try:
+        os.stat(path)
+        return True
+    except OSError as e:
+
```

Comment:
I like to leave a line above comments, otherwise I find it crowded and annoying to read.